match/lib/match/options.rb

```diff
@@ -80,6 +80,11 @@ def self.available_options
                                      description: "Renew the provisioning profiles every time you run match",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :unattended,
```

Comment:
I'm trying to think of other attribute names we can use here (`force` seems like the best choice, but it isn't worth the argument confusion). How about something like `-skip_confirmation`?  Or we could do `-force_nuke` to allow us to stick the the better `force` naming, but not overloading the actual `nuke` argument. What do you think?